pkg/gcp/client/compute.go

```diff
@@ -32,6 +32,10 @@ import (
 type ComputeClient interface {
 	// GetExternalAddresses returns a list of all external IP addresses mapped to the names of their users.
 	GetExternalAddresses(ctx context.Context, region string) (map[string][]string, error)
+	// GetVPC returns a name of VPC
+	GetVPC(ctx context.Context, network string) (*compute.Network, error)
+	// GetSubenet returns a subnet info.
+	GetSubenet(ctx context.Context, region, subnet string) (*compute.Subnetwork, error)
```

Comment:
Typo on `GetSubenet` -> `GetSubnet`

Reply:
Thanks, commit in https://github.com/gardener/gardener-extension-provider-gcp/pull/497/commits/ea08d3c63e779f44c1644aa3e6567abe3bab1c62 @kon-angelo